core/ide/src/main/java/org/alice/stageide/StoryApiConfigurationManager.java

```diff
@@ -407,18 +407,6 @@ public SwingComponentView<?> createReplacementForFieldAccessIfAppropriate(FieldA
     return null;
   }
 
-  @Override
```

Comment:
My own robot enforced pedantry, but this method removal left unused imports above.
Please remove them to make checkstyle happy.